extension/src/popup/ducks/transactionSubmission.ts

```diff
@@ -138,13 +138,15 @@ export const submitFreighterTransaction = createAsyncThunk<
         if (!res.ok) {
           return thunkApi.rejectWithValue({
             errorMessage: response,
+            response,
           });
         }
         return response;
       } catch (e) {
         const message = e instanceof Error ? e.message : JSON.stringify(e);
         return thunkApi.rejectWithValue({
           errorMessage: message,
+          response: e as any,
```

Comment:
the result code parser expects the `response` key, which will be populated with Horizon's response when this PR goes live: https://github.com/stellar/freighter-backend/pull/40

I've checked this in all permutations:
1) User has < 5.17.4 (old codebase) but new BE error format
2) user has 5.17.4 and old BE error format
3) user has 5.17.4 and new BE error format